docs/api/web-contents.md

```diff
@@ -412,6 +426,39 @@ Executes the editing command `replace` in web page.
 
 Executes the editing command `replaceMisspelling` in web page.
 
+### `webContents.findInPage(id, text[, options])`
+
+* `id` Integer
```

Comment:
It is weird to require users to provide an ID manually in API, we should generate the ID incrementally and return it in this API.